UserContext/AnonymousRequestMatcher.php

```diff
@@ -11,27 +11,27 @@
 
 namespace FOS\HttpCacheBundle\UserContext;
 
+use FOS\HttpCache\UserContext\AnonymousRequestMatcher as BaseAnonymousRequestMatcher;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\RequestMatcherInterface;
 
 /**
  * Matches anonymous requests using a list of identification headers.
  */
-class AnonymousRequestMatcher implements RequestMatcherInterface
+class AnonymousRequestMatcher extends BaseAnonymousRequestMatcher
 {
-    private $userIdentifierHeaders;
-
-    /**
-     * @param array $userIdentifierHeaders List of request headers that authenticate a non-anonymous request
-     */
-    public function __construct(array $userIdentifierHeaders)
+    public function __construct(array $options = array())
```

Comment:
we need some BC code here that detects if the options actually are the new options or a list of headers. (check for the keys that options can have and if such a key is present, its options. otherwise you need to create the options array for the parent, with the user identifier headers as the value that is set